### PR TITLE
Rebrand Dymoprint v2.4.0 to Labelle v1.0.0

### PR DESCRIPTION
**Pull request by maresb**
Attn: @tomers, @tomek-szczesny

### CODE REVIEW

**Comment by maresb** [comment]
@claui, I credited you [here](https://github.com/computerlyrik/dymoprint/blob/2ad0db6fff401b4de9f6ddbe4bc5832664bf185f/README.md#about-the-name) in the README for the name.

File: README.md [hunk 1]
```diff
@@ -115,42 +121,42 @@ It is also possible to Download a font from
...
 
 Take care of the trailing "" - you may enter text here which gets printed in front of the image
 
 ## GUI
 
-### Run DymoPrint GUI
+### Run Labelle GUI
 
-```dymoprint_gui```
+```labelle-gui```
```
Comment (tomers) [inline README.md:156]: Thank you so much for changing underscore with dash. It was rather annoying, and I planned on changing this after migration :+1:

**Review by tomers — APPROVED** [review]
Great work. Thanks @maresb !

**Comment by maresb** [comment]
This has been successfully released.
https://github.com/labelle-org/labelle/releases/tag/v1.0.0

**Pull request closed without merging**